app/withdrawal/mutations/withdraw.ts

```diff
@@ -11,37 +11,29 @@ import { Withdrawal } from '../validations'
 const withdraw = resolver.pipe(
   resolver.zod(Withdrawal),
   resolver.authorize(),
-  async ({ bank, account, amount }, ctx: Ctx) => {
-    if (!ctx.session.userId) throw new AuthorizationError()
+  async (input, { session: { userId } }: Ctx) => {
+    if (!userId) throw new AuthorizationError()
```

Comment:
```suggestion
  async (input, { session: { userId } }) => {
```

Reply:
I copied it too lol

Reply:
Fixed it in #28 but didn't touch this file as I was worried it might conflict.

Reply:
Turns out there's already a conflict 😂

Reply:
Oh nvm got it